src/material-experimental/mdc-radio/radio.ts

```diff
@@ -97,7 +97,8 @@ export class MatRadioGroup extends _MatRadioGroupBase<MatRadioButton> {
     '[class.mat-primary]': 'color === "primary"',
     '[class.mat-accent]': 'color === "accent"',
     '[class.mat-warn]': 'color === "warn"',
-    '[attr.tabindex]': 'disabled ? null : -1',
+    // Needs to be removed since it causes some a11y issues (see #21266).
+    '[attr.tabindex]': 'null',
```

Comment:
does it specifically have to be null or can it just be removed?

Reply:
It has to be null, because the tabindex can be set as `<mat-radio tabindex="2">` which will set the input, but also keep the native attribute.